xil_carma_cloud/carma_docker.launch.py

```diff
@@ -120,6 +120,14 @@ def generate_launch_description():
         description='List of String: Guidance Control Plugins that will be validated by the Guidance Plugin Validator Node if enabled'
     )
 
+    # Declare enable_opening_tunnels
+    enable_opening_tunnels = LaunchConfiguration('enable_opening_tunnels')
```

Comment:
What is this for. There is no tunnel between carma-cloud and platform. What is this configuration for.